test/acceptance/features/steps/steps.py

```diff
@@ -576,3 +576,37 @@ def update_binding_secret(context):
     output = openshift.apply(yaml.dump(secret_yaml), context.namespace.name)
     result = re.search(rf'.*{context.sb_secret}.*(created|unchanged|configured)', output)
     assert result is not None, f"Unable to apply YAML for binding secret '{context.sb_secret}': {output}"
+
+
+@then(u'cluster role "{cluster_role_name}" is available in the cluster')
+def check_cluster_role_exists(context, cluster_role_name):
+    openshift = Openshift()
+    roles_present, exit_code = openshift.is_resource_in("clusterroles")
+    if exit_code==0:
+        role_list = json.load(roles_present)
```

Comment:
`roles_present` is actually a string containig output of `kubectl get clusterroles` - see the `openshift.is_role_in` function.
Furthermore, `json.load` expects a file not a string and so this line causes the acceptance tests to [fail](https://github.com/redhat-developer/service-binding-operator/pull/979/checks?check_run_id=3042889774#step:8:123) with the following:
```
Scenario: Service Binding Operator installation should contain aggregating cluster role and are bound with Namespace.  # test/acceptance/features/addRBACForOperator.feature:2
    Given Namespace [TEST_NAMESPACE] is used                                                                             # test/acceptance/features/steps/steps.py:43
    When Service Binding Operator is running                                                                             # test/acceptance/features/steps/steps.py:83
    Then cluster role "servicebinding-controller-role" is available in the cluster                                       # test/acceptance/features/steps/steps.py:590
      Traceback (most recent call last):
        File "/home/runner/work/service-binding-operator/service-binding-operator/out/venv3/lib/python3.7/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/home/runner/work/service-binding-operator/service-binding-operator/out/venv3/lib/python3.7/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
        File "test/acceptance/features/steps/steps.py", line 595, in check_cluster_role_exists
          role_list = json.load(roles_present)
        File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/json/__init__.py", line 293, in load
          return loads(fp.read(),
      AttributeError: 'str' object has no attribute 'read'
      
      Captured stdout:
      TEST_NAMESPACE = test-namespace-772a9d798240
      ,---------,-
      | COMMAND : kubectl get ns test-namespace-772a9d798240
      '---------'-
      Namespace test-namespace-772a9d798240 is created!!!
      ,---------,-
      | COMMAND : kubectl get servicebinding
      '---------'-
      ,---------,-
      | COMMAND : kubectl get deployments --all-namespaces -o json | jq -rc '.items[] | select(.metadata.name == "service-binding-operator").metadata.namespace'
      '---------'-
      Service binding operator is running!!!
      ,---------,-
      | COMMAND : kubectl get clusterroles
      '---------'-
```